src/coprocessor/dag/mod.rs

```diff
@@ -178,6 +188,7 @@ fn handle_qe_response(
         Ok(sel_resp) => {
             let mut resp = Response::default();
             resp.set_data(box_try!(sel_resp.write_to_bytes()));
+            resp.set_can_be_cached(can_be_cached);
```

Comment:
Currently this field does not exist in `Response`. Is there a proto update?

Reply:
Yes, the proto change is in https://github.com/innerr/kvproto/tree/issue-6690, just added this field and no other changes.